lib/deep-equal.js

```diff
@@ -12,12 +15,11 @@ var isNaN = require("./is-nan");
 var isObject = require("./is-object");
 var isSet = require("./is-set");
 var isSubset = require("./is-subset");
-var getClassName = require("./get-class-name");
```

Comment:
Can we delete `get-class-name.js` then?

Reply:
Yes we can, it's only used in its test file now. I will remove them and update the branch, thanks for pointing that out 👍